schema/deploy/roles/ehs-results-exporter/grants.sql

```diff
@@ -0,0 +1,19 @@
+-- Deploy seattleflu/id3c-customizations:roles/ehs-results-exporter/grants to pg
+
+begin;
+
+revoke all on database :"DBNAME" from "ehs-results-exporter";
+revoke all on schema receiving, warehouse, shipping from "ehs-results-exporter";
+revoke all on all tables in schema receiving, warehouse, shipping from "ehs-results-exporter";
+
+grant connect on database :"DBNAME" to "ehs-results-exporter";
+
+grant usage
+    on schema shipping
+    to "ehs-results-exporter";
+
+grant select
+    on shipping.uw_reopening_ehs_reporting_v1
+    to "ehs-results-exporter";
```

Comment:
View specific grants need to live in the views.sql. Sorry this isn't very clear! 🙏 

We made the recent switch because we are dropping all views on every rework, the view specific grants will not persist once the view has been dropped. So these view grants need to be declared under the view create (similar to how we are granting "incidence-modeler" on all the views).

Reply:
Changed. This makes sense but does make things complicated. If the roles relate only to shipping views we could manage the roles in the same script as the views. That's how I did it at Xealth. Roles were at the top and the views were below them.